packages/Radio/src/Radio.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import PropTypes from "prop-types";
+import uuid from "uuid/v4";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import CheckIcon from "@paprika/icon/lib/Check";
+import radioStyles from "./Radio.styles";
+import Group from "./components/Group";
+
+const propTypes = {
+  a11yText: PropTypes.string,
+  children: PropTypes.node,
+  isChecked: PropTypes.bool,
+  isDisabled: PropTypes.bool,
+  canDeselect: PropTypes.bool,
+  onClick: PropTypes.func.isRequired,
+  size: PropTypes.oneOf(ShirtSizes.DEFAULT),
+};
+
+const defaultProps = {
+  a11yText: null,
+  isChecked: false,
+  canDeselect: false,
+  children: null,
+  isDisabled: false,
+  size: ShirtSizes.MEDIUM,
+};
+
+function Radio(props) {
+  const { a11yText, children, isChecked, canDeselect, isDisabled, size, onClick, ...moreProps } = props;
+  const radioId = React.useRef(uuid()).current;
+  const inputRef = React.useRef(null);
+
+  const styleProps = {
+    hasLabel: !!children,
+    size,
+  };
+
+  const handleKeyDown = event => {
+    if (
+      // Prevent scrolling the page with a spacerbar keypress
+      event.key === " " ||
+      // Prevent submitting forms in IE/Edge with and enter keypress
+      event.key === "Enter"
+    ) {
+      event.preventDefault();
+    }
+  };
+
+  const handleKeyUp = event => {
+    const isTriggerKey = event.key === " "; // space key
+    if (!isDisabled && isTriggerKey) {
+      onClick(event);
+    }
+  };
+
+  const inputProps = {};
+  if (a11yText) inputProps["aria-label"] = a11yText;
+  return (
+    <div data-pka-anchor="radio" css={radioStyles} {...styleProps} {...moreProps}>
+      <input
+        onChange={() => {}}
+        onClick={onClick}
+        checked={isChecked}
```

Comment:
I'm a little unsure _why_, but with this `checked` attribute, I don't see it being added to the DOM.  But if I use `defaultChecked` instead, it works as expected.  Maybe it's a React thing, like `className` or `htmlFor`... 🤷‍♂️

Reply:
![image](https://user-images.githubusercontent.com/4040102/69752962-49462b00-1107-11ea-9095-55ffeb8e4ac2.png)
Is showing for me?

Reply:
Ah yes, on initial load, the first `<input>` does have `checked`.  But when you change it, the DOM doesn't change.  I think it may be related to the other warning you were seeing:
https://github.com/acl-services/paprika/pull/261#discussion_r351460309

Reply:
It's the difference between attributes and properties described in https://github.com/facebook/react/issues/13525#issuecomment-417818906 under "DOM Properties Are Not Just "Attributes in JS". Looks like in this case React manipulates the property, not the attribute, so you see no change in the inspector.
![radio](https://user-images.githubusercontent.com/93752/69766186-ab188c00-112b-11ea-895f-6ac7e6c51789.gif)

Reply:
Yeah strange, I see what you mean now. Bit of a puzzler as to why the dom element isn't updating correctly and showing the attribute changes....

Reply:
Ok thanks @alexzherdev

Reply:
@mikrotron So i'm not sure its a problem that the dom element attributes are not updated, testing with voiceover i can still hear it announce that the radio button is selected, so I think this is fine?